Jenkinsfile_nightly

```diff
@@ -11,13 +11,62 @@ def type = "java"
 def product = "civil"
 def component = "sdt-commissioning"
 
+def secrets = [
+        'civil-${env}' : [
+//                secret('OAUTH2-CLIENT-SECRET', 'OAUTH2_CLIENT_SECRET'),
+//                secret('OAUTH2-CLIENT-ID', 'OAUTH2_CLIENT_ID'),
+//                secret('LD-SDK-KEY', 'LD_SDK_KEY')
+        ]
+]
+
+static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
+  [$class     : 'AzureKeyVaultSecret',
+   secretType : 'Secret',
+   name       : secretName,
+   version    : '',
+   envVariable: envVar
+  ]
+}
+
+def vaultOverrides = [
+        'preview' : 'aat',
+        'spreview': 'saat'
+]
+
 withNightlyPipeline(type, product, component) {
-  enableAksStagingDeployment()
-  disableLegacyDeployment()
-  nonServiceApp()
-  enableFortifyScan()
+  env.IDAM_URL = "https://idam-api.aat.platform.hmcts.net"
+  env.TEST_URL = "http://civil-sdt-commissioning-aat.service.core-compute-aat.internal"
+  env.Rules = params.SecurityRules
+  env.execution_environment = "aat"
+
+  overrideVaultEnvironments(vaultOverrides)
+  loadVaultSecrets(secrets)
+
+  enableMutationTest()
+  enableFullFunctionalTest()
+  enableSecurityScan()
+
+  after('mutationTest') {
```

Comment:
The "after" method is deprecated - see master build pipeline for message about it.  This will need to be changed to one of 'alwaysAfter', 'afterSuccess' or 'afterFailure' (most likely 'alwaysAfter').

Reply:
Addressed (in the correct PR!)